README.md

```diff
@@ -165,6 +165,26 @@ mvn -f kura/distrib/pom.xml clean install -DbuildAllContainers
 After this command runs, images can be found in your preferred container engine image list.
 
 ### Eclipse IDE
+
+> [!WARNING]
+>To make Kura work with newer versions of Eclipse, modify the lifecycle mapping by adding these XML lines
```

Comment:
Is there something we can do in our codebase to avoid impacting the end user?

Also "modify the lifecycle mapping by adding these XML lines"... what does this entails? A newcome to the project might not know what to change and where.